src/vulkan-renderer/application.cpp

```diff
@@ -217,6 +217,29 @@ void Application::load_octree_geometry() {
     m_worlds[0]->children()[6]->set_type(world::Cube::Type::EMPTY);
     m_worlds[0]->children()[7]->set_type(world::Cube::Type::EMPTY);
 
+    for (const auto &child : m_worlds[2]->children()) {
+        child->set_type(world::Cube::Type::OCTANT);
+
+        for (const auto &grandchild : child->children()) {
+            grandchild->set_type(world::Cube::Type::OCTANT);
+        }
+    }
+
+    std::shared_ptr<world::Cube> test = m_worlds[2]->children()[4]->children()[3]->children()[3];
+    test->set_type(world::Cube::Type::SOLID);
+    test->neighbor(world::Cube::NeighborAxis::X, world::Cube::NeighborDirection::POSITIVE)
+        ->set_type(world::Cube::Type::SOLID);
+    test->neighbor(world::Cube::NeighborAxis::X, world::Cube::NeighborDirection::NEGATIVE)
+        ->set_type(world::Cube::Type::SOLID);
+    test->neighbor(world::Cube::NeighborAxis::Y, world::Cube::NeighborDirection::POSITIVE)
+        ->set_type(world::Cube::Type::SOLID);
+    test->neighbor(world::Cube::NeighborAxis::Y, world::Cube::NeighborDirection::NEGATIVE)
+        ->set_type(world::Cube::Type::SOLID);
+    test->neighbor(world::Cube::NeighborAxis::Z, world::Cube::NeighborDirection::POSITIVE)
+        ->set_type(world::Cube::Type::SOLID);
+    test->neighbor(world::Cube::NeighborAxis::Z, world::Cube::NeighborDirection::NEGATIVE)
+        ->set_type(world::Cube::Type::SOLID);
+
```

Comment:
I think this is not required anymore as we have the tests and later on the random world.

Reply:
It would save me a rebase at least :sweat_smile: